db/operations.go

```diff
@@ -132,7 +136,7 @@ func getPrimaryKeyWhereClause(primaryKey map[string]string) string {
 	return strings.Join(reg[:], " AND ")
 }
 
-func prepareColValues(table *TableInfo, colValues map[string]string) (columns []string, values []string, err error) {
+func prepareColValues(table *TableInfo, colValues map[string]string, d dialect) (columns []string, values []string, err error) {
```

Comment:
Let's make the dialect the first parameter in this file.

Reply:
Done